### ISSUE TITLE
Calendar Item isSelectable() method

### ISSUE THREAD

**Issue by bkach** [MOLO17/CustomizableCalendar#14]
In the `CalendarItem` there is an `isSelectable()` method. It currently does not tell you whether the item is selectable, and instead tells you if it IS selected.

It should be renamed to `isSelected()`

**Comment by bkach** [comment]
Or is it the other way around? Does the variable `selected` need to be changed to `selectable`?

Edit: This seems to be the case. The variable `selected` should be changed to `selectable` within the `CalendarItem`

**Comment by francesco-furlan** [comment]
@bkach yep, you're right the variable `selected` should be changed to `selectable`.
I'm going to do this refactor ASAP.
